custom_components/frigate/media_source.py

```diff
@@ -468,6 +469,7 @@ def get_integration_proxy_path(self) -> str:
                 day,
                 f"{self.hour:02}",
                 self.camera,
+                timezone.replace("/", ","),
```

Comment:
Having the timezone in the path is pretty ugly, especially since "/" is a universal timezone separator. What do you think about specifying the tz as a GET parameter instead?

Reply:
I wish we could but this is an issue with the backend, nginx doesn't support this for the vod module